Package.swift

```diff
@@ -10,9 +10,14 @@
 */
 
 import PackageDescription
+import Darwin
```

Comment:
It seems we do not need to import Darwin?

Reply:
Good catch! Looks like that was automatically added by Xcode when i added the `platforms` field, but you're right, it's not necessary.